txBuilding/Backend/OgmiosChainContext/OgmiosChainContext.go

```diff
@@ -350,6 +350,10 @@ func (occ *OgmiosChainContext) AddressUtxos(address string, gather bool) []Base.
 
 }
 
+type AdaLovelace struct {
```

Comment:
Can you just use `Value` for this?

Reply:
Maybe, but I don't think there is any benefit in this case since the Base.ProtocolParameters struct just has integer fields (for lovelace).

Reply:
ok :sweat_smile: